vector-im/element-android
# Adds Push Notification toggle to Device Manager

## Type of change


## Content

Adds push notification toggle to device manager

## Motivation and context

Epic: https://element-io.atlassian.net/browse/PSG-595
Closes: https://element-io.atlassian.net/browse/PSG-779

## Screenshots / GIFs

<img width="300" src="https://user-images.githubusercontent.com/20701752/193117970-63da6d2d-a0ca-429f-b320-e0f436669594.png" />

## Tests

- Enable device manager feature flag in debug settings
- Go to security settings and select Sessions v2
- View details of a session and see that if there is a registered pusher of the device there is also a toggle

## Tested devices

- OS version(s): Android 13

## Checklist
